server.go

```diff
@@ -1131,7 +1170,9 @@ func (sp *serverPeer) OnVersion(_ *peer.Peer, msg *wire.MsgVersion) {
 		// known tip.
 		if !cfg.DisableListen && sp.server.syncManager.IsCurrent() {
 			// Get address that best matches.
-			lna := addrManager.GetBestLocalAddress(remoteAddr)
+			addrTypeFilter := natfSupported(
```

Comment:
This will fit on the same line now with the shorter func name.

Reply:
Nice, updated.